src/ol/style/expressions.js

```diff
@@ -784,3 +785,58 @@ Operators['case'] = {
     return result;
   },
 };
+Operators['in'] = {
+  getReturnType: function (args) {
+    return ValueTypes.BOOLEAN;
+  },
+  toGlsl: function (context, args) {
```

Comment:
I see that there are limited tests in place already, but it would be good to start adding tests for things like this.

Reply:
I added some unit tests for the 'in' expression.